### DISCUSSION TITLE
seeking right information

### DISCUSSION THREAD

**Discussion by Djwszone** [community/community#27731] [Other Feature Feedback, Questions, & Ideas]
Hello, my name is Dustyn Williams, im 25 and very new to all this. I am not tech savvy by any means at all unless its got something to do with motor controls or PLC's. I started this about a year ago and just now got back into my accounts but im having an issue trying to remove one of my repos, as well asdealing with a whole lot of spam and cookies. If anyone could help me better understand this so im not potentially hurting my software again please show me the correct path so i can fix my electronics. A year is far too long.

**Comment by radar-blip** [comment]
You can open the repo to be deleted. Open settings, and scroll to the bottom and delete it.

![repos](https://user-images.githubusercontent.com/39196767/181956846-7bc8384a-f7d6-4980-8a7c-5a3099044422.png)
